website/docs/GettingStarted/Documentation/01-Doc_Edits.md

```diff
@@ -0,0 +1,23 @@
+---
+id: DocEdits
+title: Editing Docs Guide 
+---
+
+This page will explain the process of contributing to the OpenBCI documentation.
+
+## Minor Changes
+
+If you'd like to make a minor change to our docs, such as updating a hyperlink or fixing a typo, all you need to do is:
+
+1. Find the page you want to edit and scroll to the bottom
+2. Click on the **Edit this page** button. You'll be redirected to your Github account.
+3. Github will ask you to fork our repository. Click on **Fork this repository**.
+3. Make the changes. 
+4. Click on **Propose changes**. 
+5. Done. Your changes will be reviewed by our admins!
+
+## Major Changes
+
+If you'd like to make a major change such as adding a new document follow the steps on our [Docs Wiki](https://github.com/OpenBCI/Documentation/wiki/Making-Changes-to-OpenBCI-Documentation). 
```

Comment:
If you'd like to make a major change, such as adding a new document, follow the steps on our

Add commas here for clarity and grammar.

Reply:
Done!